tests/unit/test_configs.py

```diff
@@ -1,156 +1,168 @@
-        _, _, req = pathname_configs()
-        self.assertEqual('/requests/', req)
-
-    def test_pathname_prefix_assets(self):
-        req = '/'
-        path = get_asset_path(req, 'reset.css', 'assets')
-        self.assertEqual('/assets/reset.css', path)
-
-        req = '/requests/'
-        path = get_asset_path(req, 'reset.css', 'assets')
-        self.assertEqual('/requests/assets/reset.css', path)
-
-        req = '/requests/routes/'
-        path = get_asset_path(req, 'reset.css', 'assets')
-        self.assertEqual('/requests/routes/assets/reset.css', path)
-
-    def test_get_combined_config_dev_tools_ui(self):
-        val1 = get_combined_config('ui', None, default=False)
-        self.assertEqual(
-            val1, False,
-            "should return the default value if None is provided for init and environment")
-        os.environ['DASH_UI'] = 'true'
-        val2 = get_combined_config('ui', None, default=False)
-        self.assertEqual(val2, True, "should return the set environment value as True")
-        val3 = get_combined_config('ui', False, default=True)
-        self.assertEqual(val3, False, "init value overrides the environment value")
-
-    def test_get_combined_config_props_check(self):
-        val1 = get_combined_config('props_check', None, default=False)
-        self.assertEqual(
-            val1, False,
-            "should return the default value if None is provided for init and environment")
-        os.environ['DASH_PROPS_CHECK'] = 'true'
-        val2 = get_combined_config('props_check', None, default=False)
-        self.assertEqual(val2, True, "should return the set environment value as True")
-        val3 = get_combined_config('props_check', False, default=True)
-        self.assertEqual(val3, False, "init value overrides the environment value")
-
-    def test_load_dash_env_vars_refects_to_os_environ(self):
-        for var in DASH_ENV_VARS.keys():
-            os.environ[var] = 'true'
-            vars = load_dash_env_vars()
-            self.assertEqual(vars[var], 'true')
-            os.environ[var] = 'false'
-            vars = load_dash_env_vars()
-            self.assertEqual(vars[var], 'false')
-
-
-@pytest.mark.parametrize('name, server, expected', [
-    (None, True, '__main__'),
-    ('test', True, 'test'),
-    ('test', False, 'test'),
-    (None, Flask('test'), 'test'),
-    ('test', Flask('other'), 'test'),
-])
-def test_app_name_server(name, server, expected):
+@pytest.mark.parametrize(
+    "name, server, expected",
+    [
+        (None, True, "__main__"),
+        ("test", True, "test"),
+        ("test", False, "test"),
+        (None, Flask("test"), "test"),
+        ("test", Flask("other"), "test"),
+    ],
+)
+def test_app_name_server(empty_environ, name, server, expected):
     app = Dash(name=name, server=server)
     assert app.config.name == expected
 
 
-if __name__ == '__main__':
-    unittest.main()
+if __name__ == "__main__":
```

Comment:
we don't need this mechanism in `pytest` anymore, we can collect and run directly with `pytest tests/unit`